crates/core/src/equivalence.rs

```diff
@@ -1,64 +1,7 @@
-use crate::binding::Binding;
 use grit_util::AstNode;
 use itertools::{EitherOrBoth, Itertools};
-use marzano_language::language::Language;
 use marzano_util::node_with_source::NodeWithSource;
 
-impl<'a> Binding<'a> {
```

Comment:
Unfortunately, a few methods like this had to be moved. This is because while `impl Type` blocks can be split into multiple blocks, `impl Trait for Type` blocks cannot.